parser_library/src/workspaces/processor_group.cpp

```diff
@@ -32,31 +35,84 @@ struct translate_pp_options
     }
 };
 
-asm_option translate_assembler_options(const config::assembler_options& asm_options)
+constexpr auto sys_arch_translator = []() {
+    constexpr std::array sys_arch_equivalents { std::make_pair(std::string_view("ZOP"), system_architecture::ZOP),
```

Comment:
wouldn't it be easier (also on the compiler) to just have a pre-sorted array? (with a static assert perhaps)

Reply:
Ok